build.gradle

```diff
@@ -23,6 +23,7 @@ repositories {
 }
 
 dependencies {
+    compile "org.slf4j:slf4j-log4j12:1.7.25"
```

Comment:
Emm, in my environment the project is crashed with exception? Did you try to start it?